daemon/src/monitor.rs

```diff
@@ -1000,6 +1018,48 @@ impl Actor {
     }
 }
 
+static LOCK_TRANSACTION_BROADCAST_COUNTER: conquer_once::Lazy<prometheus::IntCounter> =
+    conquer_once::Lazy::new(|| {
+        prometheus::register_int_counter!(
+            "blockchain_lock_transactions_broadcast_total",
+            "The number of lock transactions broadcast."
+        )
+        .unwrap()
+    });
+static COMMIT_TRANSACTION_BROADCAST_COUNTER: conquer_once::Lazy<prometheus::IntCounter> =
+    conquer_once::Lazy::new(|| {
+        prometheus::register_int_counter!(
+            "blockchain_commit_transactions_broadcast_total",
+            "The number of commit transactions broadcast."
+        )
+        .unwrap()
+    });
+static REFUND_TRANSACTION_BROADCAST_COUNTER: conquer_once::Lazy<prometheus::IntCounter> =
+    conquer_once::Lazy::new(|| {
+        prometheus::register_int_counter!(
+            "blockchain_refund_transactions_broadcast_total",
+            "The number of refund transactions broadcast."
+        )
+        .unwrap()
+    });
+static COLLABORATIVE_CLOSE_TRANSACTION_BROADCAST_COUNTER: conquer_once::Lazy<
```

Comment:
❓ Wasn't it collaborative settlement? Nitty, but it would be good to use consistent terminology.

Reply:
Uhm, this file calls it collaborative close :sweat_smile: